lib/config/configuration.go

```diff
@@ -1754,6 +1754,21 @@ kubernetes matchers are present`)
 				AssumeRole: assumeRole,
 			})
 		}
+		for _, azureMatcher := range fc.Discovery.AccessGraph.Azure {
+			regions := azureMatcher.Regions
+			if len(regions) == 0 {
+				return trace.BadParameter("missing regions in access_graph.azure")
+			}
+			subscriptionID := azureMatcher.SubscriptionID
```

Comment:
do we support wildcard for subscription?

Reply:
No, but will add support for this.

Reply:
https://github.com/gravitational/access-graph/issues/1376